components/x11/tigervnc/Makefile

```diff
@@ -17,6 +17,7 @@ BUILD_BITS= 64
 BUILD_STYLE= cmake
 
 include ../../../make-rules/shared-macros.mk
+include $(WS_MAKE_RULES)/x11.mk
 
 COMPONENT_NAME= tigervnc
 COMPONENT_VERSION= 1.8.0
```

Comment:
Don't you want to change COMPONENT_REVISION?

Reply:
Fixed, thank you!